openverse_catalog/dags/providers/provider_api_scripts/phylopic.py

```diff
@@ -23,202 +23,114 @@
-            pub_date = result.get("submitted").strip()
-
-        return creator, credit_line, pub_date
+        TODO: Adapt `url` and `creator_url` to avoid redirects.
+        """
 
-    def get_record_data(self, data: dict) -> dict | list[dict] | None:
-        uid = data.get("uid")
+        uid = data.get("uuid")
         if not uid:
-            return
-        logger.debug(f"Processing UUID: {uid}")
-        params = {
-            "options": " ".join(
-                [
-                    "credit",
-                    "licenseURL",
-                    "pngFiles",
-                    "submitted",
-                    "submitter",
-                    "taxa",
-                    "canonicalName",
-                    "string",
-                    "firstName",
-                    "lastName",
-                ]
-            )
-        }
-        endpoint = f"{self.base_endpoint}/{uid}"
-        response_json = self.get_response_json(params, endpoint)
-        result = self._get_response_data(response_json)
-        if not result:
             return None
 
-        meta_data = {}
-        uid = result.get("uid")
-        license_url = result.get("licenseURL")
-
-        img_url, width, height = self._get_image_info(result, uid)
-
-        if img_url is None:
+        data = data.get("_links", {})
+        license_url = data.get("license", {}).get("href")
+        img_url = data.get("sourceFile", {}).get("href")
+        foreign_url = data.get("self", {}).get("href")
+        if not license_url or not img_url or not foreign_url:
             return None
 
-        meta_data["taxa"], title = self._get_taxa_details(result)
+        foreign_url = self.host + foreign_url
 
-        foreign_url = self._image_url(uid)
-
-        (
-            creator,
-            meta_data["credit_line"],
-            meta_data["pub_date"],
-        ) = self._get_creator_details(result)
+        title = data.get("self", {}).get("title")
+        creator = data.get("contributor", {}).get("title")
+        creator_url = self.host + data.get("contributor", {}).get("href")
+        width, height = self._get_image_sizes(data)
 
         return {
+            "license_info": get_license_info(license_url=license_url),
             "foreign_identifier": uid,
             "foreign_landing_url": foreign_url,
             "image_url": img_url,
-            "license_info": get_license_info(license_url=license_url),
+            "title": title,
+            "creator": creator,
+            "creator_url": creator_url,
             "width": width,
             "height": height,
-            "creator": creator,
-            "title": title,
-            "meta_data": meta_data,
+            # TODO: Evaluate whether to include upstream thumbnails.
+            # Sizes available: 192x192, 128x128, 64x64.
+            # "thumbnail": thumbnail,
+            # TODO: Evaluate whether to include nodes' titles as tags.
+            # "tags": tags,
```

Comment:
I think we could re-use the Taxonomy code that @rwidom wrote for iNaturalist here. The titles in phylopic are the Latin names, right?
Not for this PR to implement, but maybe a TODO comment we could add here?

Reply:
I'm not sure the code is reusable in this case because iNaturalist is processed in SQL but the logic could be similar.

Reply:
Oh wow, I haven't seen the iNaturalist code before! It's awesome (and I'm afraid I wouldn't be able to use something similar here). No TODO then :) 
But it would be nice to open an issue for adding Vernacular names to Phylopic metadata, to make the images more discoverable.